### PR TITLE
create image & navigation components

### PR DESCRIPTION
**Pull request by agrawalshanu708**
image component: created round image & responsive image
navigation component: created navbar

### CODE REVIEW

**Review by tanishk15sharma — APPROVED** [review]

File: image/image.css [hunk 1]
```diff
@@ -0,0 +1,12 @@
+@import "../resetcss.css";
```
Comment (yashpatel1797) [inline image/image.css:1]: you can name this file as "reset.css" only

File: image/image.css [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+.round-img{
+    border-radius: 50%;
+    width: 20rem;
+}
+.responsive-img{
+    width: 100%;
+    max-width: 40rem;
+    height: auto;
+    width: 60rem;
+    height: 40rem;
```
Comment (yashpatel1797) [inline image/image.css:11]: width and height are declared two times remove one property.

File: image/image.html [hunk 3]
```diff
@@ -0,0 +1,121 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```
Comment (yashpatel1797) [inline image/image.html:6]: add author meta tag.

**Review by yashpatel1797 — APPROVED** [review]
Code looks good to merge.

File: image/image.html (hunk 3 above)
Comment (agrawalshanu708) [reply image/image.html:6]: thnx yashpatel for reviewing

**Pull request merged**